src/action/common/configure_enterprise_edition_init_service.rs

```diff
@@ -0,0 +1,188 @@
+use std::path::PathBuf;
+
+#[cfg(target_os = "macos")]
+use serde::{Deserialize, Serialize};
+#[cfg(target_os = "macos")]
+use tokio::io::AsyncWriteExt;
+use tokio::process::Command;
+use tracing::{span, Span};
+
+use crate::action::{ActionError, ActionErrorKind, ActionTag, StatefulAction};
+use crate::execute_command;
+
+use crate::action::{Action, ActionDescription};
+
+#[cfg(target_os = "macos")]
+const DARWIN_ENTERPRISE_EDITION_DAEMON_DEST: &str =
+    "/Library/LaunchDaemons/systems.determinate.nix-daemon.plist";
+/**
+Configure the init to run the Nix daemon
+*/
+#[derive(Debug, serde::Deserialize, serde::Serialize, Clone)]
+pub struct ConfigureEnterpriseEditionInitService {
+    start_daemon: bool,
+}
+
+impl ConfigureEnterpriseEditionInitService {
+    #[tracing::instrument(level = "debug", skip_all)]
+    pub async fn plan(start_daemon: bool) -> Result<StatefulAction<Self>, ActionError> {
+        Ok(Self { start_daemon }.into())
```

Comment:
These EE-specific steps should probably ensure that the "enablement" binary is present, or otherwise mark it as `Skipped` (or error). Additionally, this seems to be causing some issues in CI where it just.... hangs at some point (why is the CI test trying to enable EE mode? surely that's wrong). That hang will probably rear its ugly head outside of CI too.

Reply:
I had accidentally inverted the bool in the planner:  https://github.com/DeterminateSystems/nix-installer/pull/905/commits/045f388c2f17b51f6c1c178912eaa935d26698ef

The root macos planner does make the check that the binary is present. I'm not sure we should duplicate the behavior in the actions ... what do you think?